src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -104,7 +104,7 @@
                               :counter="MAX_RESOLUTION_TEXT_LENGTH"
                               v-model="resolution"
                               :rules="resolutionRules"
-                              @change="onresolutionChanged"
+                              @change="onResolutionChanged"
```

Comment:
This works but I prefer to be explicit that this is a function call, eg.
`@change="onResolutionChanges($event)`

Doesn't _have_ to be changed, but if you then please also update lines 88, 173, etc.